test/gleam/string_test.gleam

```diff
@@ -415,6 +415,11 @@ pub fn utf_codepoint_test() {
 
   string.utf_codepoint(55296)
   |> should.be_error
+
+  assert Ok(star) = string.utf_codepoint(127775)
+  star
+  |> string.utf_codepoint_to_int
```

Comment:
I think there should be a separate test function for this separate string modue pub function to test AFAIR